dsl/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        minSdkVersion 14
```

Comment:
Would be great to share these constants between modules and have them in a single file for easier/safer updates in future